docs/src/snap/howto/epa.md

```diff
@@ -0,0 +1,1061 @@
+  --reserved-cpus: "0-31"
+  --cpu-manager-policy: "static"
+  --topology-manager-policy: "best-effort"
+```
+
+3. Bootstrap {{product}} using the above configuration file.
+
+```
+sudo k8s bootstrap --file configuration.yaml
+```
+
+#### Verify control plane node is up
+
+After a few seconds you can query the API server with:
+
+```
+sudo k8s kubectl get all -A
+```
+
+### Second k8s node as worker 
+
+1. Install the k8s snap on the second node
+
+```
+sudo snap install k8s --classic --channel=1.30-moonray/beta
+```
+
+2. On the control plane node generate a join token to be used for joining the
+   second node
+
+```
+sudo k8s get-join-token --worker
+```
+
+3. On the worker node create the configuration.yaml file
+
+```
+extra-node-kubelet-args:
+  --reserved-cpus: "0-31"
+  --cpu-manager-policy: "static"
+  --topology-manager-policy: "best-effort"
+```
+
+4. On the worker node use the token to join the cluster
+
+```
+sudo k8s join-cluster --file configuration.yaml <token-generated-on-the-control-plane-node>
+```
+
+
+### Verify the two node cluster is ready 
+
+After a few seconds the second worker node will register with the control
+plane. You can query the available workers from the first node:
+
+```
+sudo k8s kubectl get nodes
+```
+
+The output should list the connected nodes:
+
+```
+NAME          STATUS   ROLES                  AGE   VERSION
+pc6b-rb4-n1   Ready    control-plane,worker   22h   v1.30.2
+pc6b-rb4-n3   Ready    worker                 22h   v1.30.2
+```
+
+### Multus and SRIOV setup 
+
+Get the thick plugin (in case of resource scarcity we can consider deploying
+the thin flavor)
+
+```
+sudo k8s kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/master/deployments/multus-daemonset-thick.yml
+```
+
+Note: the memory limits for the multus pod spec in the DaemonSet should be
+increased (i.e. to 500Mi instead 50Mi) to avoid OOM issues when deploying
+multiple workload pods in parallel.
+
+#### SRIOV Network Device Plugin 
+
+* Create sriov-dp.yaml configMap
+
+```
+cat <<EOF | tee sriov-dp.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+ name: sriovdp-config
+ namespace: kube-system
+data:
+ config.json: |
+   {
+       "resourceList": [{
+               "resourceName": "intel_sriov_netdevice",
+               "selectors": [{
+                   "vendors": ["8086"],
+                   "devices": ["1889"],
+                   "drivers": ["iavf"]
+               }]
+           },
+           {
+               "resourceName": "intel_sriov_dpdk",
+               "resourcePrefix": "intel.com",
+               "selectors": [{
+                   "vendors": ["8086"],
+                   "devices": ["1889"],
+                   "drivers": ["vfio-pci"],
+                   "pfNames": ["enp152s0f1"],
+                   "needVhostNet": true
+               }]
+           }
+       ]
+   }
+EOF
+```
+
+Apply the configMap definition:
+
+```
+sudo k8s kubectl create -f ./sriov-dp.yaml
```

Comment:
We could also apply this from a gh raw file (as below) unless we want the user to take a second look at the config?

Reply:
why don't we make a card to find all the long blocks of code in all docs and make them downloadable.